fili-core/src/test/groovy/com/yahoo/bard/webservice/web/SlicesApiRequestSpec.groovy

```diff
@@ -94,9 +94,10 @@ class SlicesApiRequestSpec extends BaseDataSourceMetadataSpec {
         table.allAvailableIntervals.each {
             Map<String, Object> row = [:] as LinkedHashMap
             row["intervals"] = it.value
-            row["name"] = table.getPhysicalColumnName(it.key.name)
+            row["name"] = it.key.name
             if (it.key instanceof DimensionColumn) {
                 row["uri"] = uri + it.key.name
+                row["factName"] = table.getPhysicalColumnName(it.key.name)
```

Comment:
Just curious, why are we putting this inside a conditional?

Reply:
Because `factName` was valid only for dimension rows